pkgs/misc/emulators/dolphin-emu/default.nix

```diff
@@ -15,17 +15,21 @@ stdenv.mkDerivation rec {
     sha256 = "07mlfnh0hwvk6xarcg315x7z2j0qbg9g7cm040df9c8psiahc3g6";
   };
 
+  patches = [
+    ./5.0-soundtouch-exception-fix.patch
```

Comment:
Can we use `fetchpatch` from
* https://src.fedoraproject.org/rpms/dolphin-emu/raw/a1b91fdf94981e12c8889a02cba0ec2267d0f303/f/dolphin-emu-5.0-soundtouch-exception-fix.patch

(url is stable)